runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -84,15 +86,15 @@ class SkillCalculator extends JPanel
 	private final List<UIActionSlot> uiActionSlots = new ArrayList<>();
 	private final UICombinedActionSlot combinedActionSlot;
 	private final ArrayList<UIActionSlot> combinedActionSlots = new ArrayList<>();
-	private final List<JCheckBox> bonusCheckBoxes = new ArrayList<>();
+	private final Map<SkillBonus, JCheckBox> bonusCheckBoxes = new HashMap<>();
 	private final IconTextField searchBar = new IconTextField();
 
 	private CalculatorType currentCalculator;
 	private int currentLevel = 1;
 	private int currentXP = Experience.getXpForLevel(currentLevel);
 	private int targetLevel = currentLevel + 1;
 	private int targetXP = Experience.getXpForLevel(targetLevel);
-	private SkillBonus currentBonus = null;
+	private final List<SkillBonus> currentBonus = new ArrayList<>();
```

Comment:
`currentBonus` -> `currentBonuses`. Also a `Set` rather than `List` perhaps, since uniqueness should be guaranteed, and ordering is not important.

Reply:
Would I just do 
```
private final Set<SkillBonus> currentBonus = HashSet.newHashSet(0);
```
since 
```
private final Set<SkillBonus> currentBonus = EnumSet.noneOf(SkillBonus.class);```
isn't possible?